pkgs/applications/graphics/kodelife/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchzip
+, alsaLib
+, glib
+, gst_all_1
+, libGLU_combined
+, xlibs
```

Comment:
Clicking through to Details on the aliases check, we can see:

> anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/eval-0-gleber.ewr1.nix.ci/pkgs/applications/graphics/kodelife/default.nix:1:1 called without required argument 'xlibs', at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/eval-0-gleber.ewr1.nix.ci/lib/customisation.nix:69:12

 specifically, 

> called without required argument 'xlibs'

indicating `xlibs` is an alias: https://github.com/NixOS/nixpkgs/blob/6576162fe9e6ddaf1df95429b8462e84a2ef1c06/pkgs/top-level/aliases.nix#L387

Reply:
Fixed in 8492dfeaba2569f3eba6e910ecd970d7ed45e028